src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -972,6 +1063,26 @@ private
   rewrite *-comm-≡ r p
         | *-comm-≡ r q = *-cancelʳ-≤-pos r>0
 
+*-cancelʳ-≤-neg : ∀ {r} → Negative r → ∀ {p q} → p * r ≤ q * r → q ≤ p
+*-cancelʳ-≤-neg {r} r<0 {p} {q} pr≤qr = neg-cancel-≤ (*-cancelʳ-≤-pos (positive { - r} (neg-mono-< {r} {0ℚᵘ} (negative⁻¹ r<0))) (begin
+  - p * - r                                ≈˘⟨ neg-distribˡ-* p (- r) ⟩
+  - (p * - r)                              ≈˘⟨ -‿cong (neg-distribʳ-* p r) ⟩
+  - - (p * r)                              ≈⟨ neg-involutive (p * r) ⟩
+  p * r                                    ≤⟨ pr≤qr ⟩
+  q * r                                    ≈˘⟨ neg-involutive (q * r) ⟩
+  - - (q * r)                              ≈⟨ -‿cong (neg-distribʳ-* q r) ⟩
+  - (q * - r)                              ≈⟨ neg-distribˡ-* q (- r) ⟩
+  - q * - r                                ∎))
```

Comment:
I don't think this "global file alignment" of equational reasoning works very well. Firstly it's very brittle (make one line longer somewhere and you have to adjust all the proofs) and secondly it's hard to read along the line. Equational reasoning should be aligned on a proof by proof basis, with 2 spaces in between the longest term and the start of the reasoning.